File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.js
function solution(num_list) {
    let answer = 0;
    for (let i = 0; i < num_list.length; i++) {
        let num = num_list[i];
        while (num != 1) {
             if (num % 2 != 0) {
                 num -= 1;
             } else {
                 num = num / 2;
                 answer++;
             }
        }
    }
    return answer;
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.py
def solution(num_list):
    answer = 0
    for num_item in num_list:
        num = num_item
        while num != 1:
            if num % 2 != 0:
                num -= 1
            else:
                num = num // 2
                answer += 1
    return answer